sbc_compassion/models/correspondence.py

```diff
@@ -487,9 +488,11 @@ def create(self, vals):
         if vals.get("store_letter_image", True) is False:
             vals["letter_image"] = False
 
+        contract = self.env["recurring.contract"].browse(vals["sponsorship_id"])
+        contract.last_sponsor_letter = datetime.now()
```

Comment:
We only want to mark supporter letters and not update the date when we receive a child letter.
```suggestion
        contract = self.env["recurring.contract"].browse(vals["sponsorship_id"])
        if vals.get("direction") == "Supporter To Beneficiary":
            contract.last_sponsor_letter = fields.Date.today()
```